config/software/openssl3.rb

```diff
@@ -89,8 +90,6 @@
 
   command configure_command, env: env, in_msys_bash: true
 
-  patch source: "openssl-3.0.9-do-not-build-docs.patch", env: env
```

Comment:
What is the reasoning for this removal?

Reply:
I haven't checked but I suppose the patch doesn't apply anymore. I will double check

Reply:
I can't see an obvious reason not to apply the patch, I restored it and will check the pipeline

Reply:
I assumed openssl's Makefile was versionned but it isn't :facepalm: I opened a new [PR](https://github.com/DataDog/omnibus-software/pull/532) to have the same behavior regardless of the openssl version and will rebase this PR on top of it